app/controllers/merchant_items_controller.rb

```diff
@@ -8,4 +8,27 @@ def show
     @merchant = Merchant.find(params[:merchant_id])
     @item = @merchant.items.find(params[:id])
   end
+
+  def edit
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = @merchant.items.find(params[:id])
+  end
+
+  def update
+    merchant = Merchant.find(params[:merchant_id])
+    item = merchant.items.find(params[:id])
```

Comment:
If having an `item_params` method is a security thing to keep information separate, does calling `params` expose us to those potential security threats?

(I genuinely don't know the answer to this question or what the right thing to do is)

Reply:
uhhhh I also don't know